rosdep/base.yaml

```diff
@@ -3051,6 +3051,16 @@ libnl-dev:
   fedora: [libnl-devel]
   gentoo: [dev-libs/libnl]
   ubuntu: [libnl-dev]
+libnlopt-cxx-dev:
+  debian:
+    '*': [libnlopt-cxx-dev]
+    stretch: null
+  fedora: [NLopt-devel]
+  gentoo: [sci-libs/nlopt]
+  ubuntu:
+    '*': [libnlopt-cxx-dev]
```

Comment:
@mgruhler isn't the package available starting with `eoan`?

Reply:
It is. My understanding is that this rule will work for all distros except the `null` ones (i.e. bionic and focal).
`eoan` is EOL'd anyways...

Reply:
> My understanding is that this rule will work for all distros except the `null` ones 

Correct, even for older ones where it does not apply. Perhaps we should be specific if not too verbose.

Reply:
as above:

```yaml
ubuntu:
  eoan: [libnlopt-cxx-dev]  # well, this is EOL. Should we still add it?
  focal: [libnlopt-cxx-dev]
```

Reply:
> `eoan: [libnlopt-cxx-dev]`  # well, this is EOL. Should we still add it?

Yes.

Reply:
@dirk-thomas thanks for the feedback. I'm not quite sure though, what you want me to do. Should I 

1. keep it the way it is (wildcard set and thus available for `eoan`, but excluding ROS-supported LTS Distros `bionic` and `xenial`, where it does not exist, and effectively ignoring all non-LTS older then eoan) or 
1. change it to be specific (i.e. set explicitely for `eoan` and `focal` and ignore all others, effectively adding the "constant maintanence burden" you mention above)?

Reply:
Option 1 is sufficient. Non-EOL distros (like option 2) isn't usually something we require for new keys. So it can stay as is imo.

Reply:
Great, thx.